packages/react-native-bridge/android/src/main/java/org/wordpress/mobile/FileUtils.java

```diff
@@ -35,4 +37,19 @@ public static String getStringFromInputStream(InputStream inputStream) throws IO
         }
         return sb.toString();
     }
+
+    public static List<String> getAssetFileList(Activity activity, String path) {
+        ArrayList<String> pathList = new ArrayList<>();
+        try {
+            String [] list = activity.getAssets().list(path);
+            if (list.length > 0) {
```

Comment:
Do we need this length check? If the list is empty, then the following for-loop on line 46 won't do anything, which has the same effect as skipping it if the length is not greater than 0.

Reply:
Yup, thanks. Fixed.